pkg/volume/glusterfs/glusterfs.go

```diff
@@ -231,9 +231,41 @@ func (b *glusterfsMounter) SetUpAt(dir string, fsGroup *int64) error {
 	return err
 }
 
+// GlusterFsMountErrorHint performs some basic analysis
+// on the current mount error returned from the plugin
+// and will add a user hint or resolution tip for enhanced UXP
+// If no matches then original error is returned
+func GlusterFsMountErrorHint (inErr error) error {
+	if inErr == nil {
+		return nil
+	}
+	if dstrings.Contains(inErr.Error(), "lstat") && dstrings.Contains(inErr.Error(), "permission denied") {
+		return fmt.Errorf("%v\n\nAdditional Info: The pod is running, and the mount succeeded, however the mount is not accessible due to permissions.\nCheck the POSIX based permissions (owner, groups and others) on your mounted directory.\nIf needed, containers and pods can utilize and pass in a securityContext specifying runAsUser (uid/owner), or additional linux groups such as fsGroup (for block) or SupplementalGroups (for shared).\nWork with the storage adminstrator to ensure correct volume access.\n", inErr)
```

Comment:
I would avoid saying the pod is running here.  That seems likely to be misleading -- you do not know that at all from this call site.